src/script/miniscript.h

```diff
@@ -344,13 +349,32 @@ struct Ops {
     Ops(uint32_t in_count, MaxInt<uint32_t> in_sat, MaxInt<uint32_t> in_dsat) : count(in_count), sat(in_sat), dsat(in_dsat) {};
 };
 
+struct SatInfo {
+    //! Maximum witness size to (dis)satisfy;
+    MaxInt<uint32_t> size;
```

Comment:
In 80606412f087eef74dedea008dc84a5efb715a49 "miniscript: check maximum stack size during execution"

Is this size as in bytes or size as in number of items on the stack?

Reply:
This is the size of the witness stack (since we want to check the stack size during execution). I'll update the comment to make it more explicit.

Reply:
Added a comment to clarify.